app/views/_controls.html.erb

```diff
@@ -0,0 +1,26 @@
+<!-- copied from https://raw.githubusercontent.com/samvera/hyrax/main/app/views/_controls.html.erb -->
+
+<nav class="navbar navbar-default navbar-static-top" role="navigation" aria-label="Root Menu">
+  <div class="container-fluid">
+    <div class="row">
+      <ul class="nav navbar-nav col-sm-5">
+        <li <%= 'class=active' if current_page?(hyrax.root_path) %>>
```

Comment:
There is a lot of repeated code here. Do you think we can make a helper method that does all this `li` and `link_to` configuration? Let me know if you  want to pair!

Reply:
This is another file copied from hyrax which i think should be left as close to the original as possible for easy comparing of both files in future if things deviate.